python/check_hex_string.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# Copyright 2017 Daniel Estevez <daniel@destevez.net>
```

Comment:
The copyright statement needs to be updated to the year and author.

Reply:
done.